sound/soc/sof/sof-audio.c

```diff
@@ -725,7 +725,7 @@ int sof_tear_down_pipelines(struct snd_sof_dev *sdev, bool verify)
 	 * sroute->setup because during suspend all streams are suspended and during topology
 	 * loading the sound card unavailable to open PCMs.
 	 */
-	list_for_each_entry_reverse(swidget, &sdev->widget_list, list) {
+	list_for_each_entry(swidget, &sdev->widget_list, list) {
```

Comment:
I'd keep the reverse here as well to make sure things dom't break in older FW

Reply:
added explicit dependency.